src/Http/Middleware/ForbidBannedUser.php

```diff
@@ -47,7 +47,7 @@ public function handle($request, Closure $next)
         $user = $this->auth->user();
 
         if ($user && $user instanceof BannableContract && $user->isBanned()) {
-            return redirect()->back()->withInput()->withErrors([
+            return redirect()->route(config('ban.route', 'login'))->withInput()->withErrors([
```

Comment:
This change might lead to recursion if login page redirects logged in user, then he will be redirected back to login and then redirected forward and then back again.